### PR TITLE
fix nuclei #8920

### PR DESCRIPTION
**Pull request by manuel-sommer**
see issue https://github.com/DefectDojo/django-DefectDojo/issues/8920

### CODE REVIEW

**Review by cneill — APPROVED** [review]

File: dojo/tools/nuclei/parser.py
Comment (mtesauro) [inline dojo/tools/nuclei/parser.py:1]: @manuel-sommer You can ignore these messages from DryRunSecurity App - I'm working on how to suppress them. Sorry.

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by manuel-sommer** [comment]
@Maffooch and @dsever please approve this MR

**Review by Maffooch — APPROVED** [review]

**Comment by manuel-sommer** [comment]
@grendel513 please approve and merge :-)

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**